### ISSUE TITLE
Not able to login on OC client - but user works fine in browser

### ISSUE THREAD

**Issue by wioxjk** [owncloud/client#5373]
### Expected behaviour
I expected a successful login from the OC client (Windows Version 2.2.4 build 6408) to my newly deployed OC server

### Actual behaviour
I am unable to login with the OC client (Windows Version 2.2.4 build 6408)
Access forbidden by server. To verify that you have proper access, click here to access the service with your browser.

### Steps to reproduce
1. Deploy as fresh install with Owncloud Server
2. Install the latest version of OC client
3. Try to login via HTTPS with your OC client

### Server configuration
Operating system:
Debian 8
Web server: 
Apache 2.4
Database:
mySQL
PHP version:
5.6-fpm
ownCloud version:
9.1.2

### Client configuration
Client version:
Version 2.2.4 build 6408

Operating system:
Windows 10

OS language:
English

Installation path of client:
C:\Program Files (x86)\ownCloud (default)

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Client logfile: Output of `owncloud --logwindow` or `owncloud --logfile log.txt`
12-09 23:21:33:619 0x909c58 OCC::OwncloudSetupWizard::slotConnectToOCUrl: Connect to url:  "https://owncloud.domain.se"
12-09 23:21:33:620 0x909c58 OCC::OwncloudWizard::appendToConfigurationLog: Setup-Log:  "Trying to connect to ownCloud at https://owncloud.domain.se..."
12-09 23:21:33:620 0x909c58 OCC::AbstractNetworkJob::start: !!! OCC::PropfindJob created for "https://owncloud.domain.se" + "/" "OCC::OwncloudSetupWizard"
12-09 23:21:33:747 0x909c58 OCC::HttpCredentials::slotAuthentication: Stop request: Authentication failed for  "https://owncloud.domain.se/remote.php/webdav/"
12-09 23:21:33:747 0x909c58 OCC::AbstractNetworkJob::slotFinished: void OCC::AbstractNetworkJob::slotFinished() 5 "Operation canceled" QVariant(Invalid)
12-09 23:21:33:748 0x909c58 OCC::PropfindJob::finished: PROPFIND request *not* successful, http result code is 0 ""

2. Web server error log:
Nothing generated when trying to login

3. Server logfile: ownCloud log (data/owncloud.log):
Nothing generated  when trying to login

**Comment by wioxjk** [comment]
Any idea what the issue can be?

**Comment by RealRancor** [comment]
@wioxjk I suggest to close here and post your issue at https://central.owncloud.org/ where such support requests belongs to (especially as you're running php-fpm).

There you will also find FAQs like https://central.owncloud.org/t/how-to-fix-caldav-carddav-webdav-problems/852 supporting you to fix your environment.

**Comment by wioxjk** [comment]
@RealRancor 
Actually, I had the same problem even if I installed OC from repository too. And that is not using fpm.
I will post the issue over there but I will leave this open - and post the reponse/solution here

**Comment by RealRancor** [comment]
> I will post the issue over there but I will leave this open 

Please make sure to work through the linked FAQ before. Especially the "Authentication fails / special characters" is also important.

**Comment by wioxjk** [comment]
@RealRancor 
I have done that already, nothing works. I have installed OC 15-20 times before but this time it just does'nt work. And that is making me sleep bad at night

**Comment by wioxjk** [comment]
I have not got any reply on my post on central.owncloud.org.
And after a patch, I still got the same error, any solutions yet?
If not - I am force to look after other software

**Comment by RealRancor** [comment]
@wioxjk You mostly can't expect a solution for environmental issues like yours in here. Stay on central or switch from Apache + php-fpm to:

Apache + mod_php (official supported)
nginx + php-fpm (community supported)

**Comment by SamuAlfageme** [comment]
Hey @wioxjk from the logs I can't really tell where the problem is (can be a firewall error, proxy settings or more likely to do with some server config). If you don't mind to provide me a dummy test account in your server, I can try to tackle down the issue. (You can see my mail in my github profile 😉)

**Comment by RealRancor** [comment]
@SamuAlfageme Please continue with giving support at https://central.owncloud.org/t/not-able-to-login-on-oc-client-but-user-works-fine-in-browser/4491

The issue here is to 99% caused by a misconfiguration on server side related to Apache + php-fpm so further support doesn't belong in the client issue tracker.

**Comment by SamuAlfageme** [comment]
@RealRancor done, closing here.

**Comment by RealRancor** [comment]
Thanks. Short hint: https://github.com/owncloud/core/issues/7719 might be related

Overall the following is still highly advised:

> switch from Apache + php-fpm to:
>
> Apache + mod_php (official supported)
> nginx + php-fpm (community supported)

**Comment by wioxjk** [comment]
As I wrote in the github issue, I have installed OC server with Vultr's image. Also on with the repository on a completly new server without php5-fpm and just php-cgi together with apache.

I have also tried installing it with the php-script provided by owncloud, and last I have tried to install it by downloading the files and uploading them to the server. Everytime I have tried installing it, I have done it from scratch.

I have tried installing with and without fpm, with php 5.6 and 7.0 but the problem persist. That is why I suspect that the issue is client related and not serverrelated.
Note that I have tried connect from 5-6 different clients on different OS, but problem is still the same.
